announcement/models/announcement_tag.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class AnnouncementTag(models.Model):
+    _name = "announcement.tag"
+    _description = "Announcement Tags"
+
+    name = fields.Char(required=True, translate=True)
+    color = fields.Integer()
+    parent_id = fields.Many2one(
+        comodel_name="announcement.tag",
+        string="Parent Tag",
+        index=True,
+        ondelete="cascade",
+    )
+    full_name = fields.Char(compute="_compute_full_name")
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        string="Company",
+        required=True,
+        index=True,
+        default=lambda self: self.env.user.company_id,
```

Comment:
The other was the way pre-v13

```suggestion
        default=lambda self: self.env.company,
```

https://www.odoo.com/documentation/15.0/es/developer/howtos/company.html